src/backends/custom/custom_backend.cc

```diff
@@ -329,6 +329,12 @@ CustomBackend::Context::Run(
   LOG_VERBOSE(1) << "Running " << name_ << " with " << requests.size()
                  << " requests";
 
+#ifdef TRTIS_ENABLE_STATS
```

Comment:
It would be nice to just make a macro for this so these can turn into 1 line:

INFER_STATS_SET_TIMESTAMP(compute_start_ns)

A little strange as you also need to declare the variable inside the macro. Perhaps make 2 macros, one that declares the variable and sets it and one that just sets it:

INFER_STATS_DECL_TIMESTAMP(compute_start_ns)

and

INFER_STATS_SET_TIMESTAMP(compute_start_ns)